packages/components/date-picker/src/date-picker-com/panel-month-range.vue

```diff
@@ -184,6 +184,7 @@ const handleRangePick = (val: RangePickValue, close = true) => {
   if (maxDate.value === maxDate_ && minDate.value === minDate_) {
     return
   }
+  emit('calendar-change', [minDate_.toDate(), maxDate_ && maxDate_.toDate()])
```

Comment:
```suggestion
  emit('pick', [minDate_.toDate(), maxDate_ && maxDate_.toDate()])
```

Reply:
for event 'pick', finally it turns to model value change, which conflict its original logic(skip the validate);

Reply:
for 'calendar-change', I just need a event to get the intermediate picked value, it's enough and doesnt bother any other, like date-range-picker

Reply:
or i misunderstood?

Reply:
You are right, I seem to be mistaken.